app/main.go

```diff
@@ -160,7 +160,12 @@ func main() {
 	}
 	defer store.Close()
 
-	h := web.NewHandler(config, metrics, store)
+	metricCfg, err := tsbridge.NewMetricConfig(context.Background(), config, store)
```

Comment:
We should be using a single context for the entire program, and passing that context around.

```
ctx, cancel := context.WithCancel(context.Background())
metricCfg, err := tsbridge.NewMetricConfig(ctx, config, store)
```

Could you also update the CreateMetrics() and the syncLoop() calls, and re-use the same context there?